app/_landing_pages/gateway/openid-connect.yaml

```
@@ -0,0 +1,293 @@
+                  example: "[Refresh Token grant example](/plugins/openid-connect/examples/refresh-token/)"
+                  howto: "[OpenID Connect and Keycloak with the client credentials grant](/how-to/configure-oidc-with-refresh-token/)"
+                - auth: Password grant
+                  description: |
+                    With the Password grant, clients can get an access token by providing a username and password to an IdP.
+                    <br><br>
+                    [Workflow diagram &rarr;](/plugins/openid-connect/#password-grant-workflow)
+                  example: "[Password grant example](/plugins/openid-connect/examples/password/)"
+                  howto: "[OpenID Connect and Keycloak with the password grant](/how-to/configure-oidc-with-password-grant/)"
+  
+  - header:
+      type: h2
+      text: "Authorization methods"
+    columns:
+      - blocks:
+          - type: table
+            config:
+              columns:
+                - title: Authorization method
+                  key: auth
+                - title: Description
+                  key: description
+                - title: Examples in every supported tool
+                  key: example
+                - title: How-to guide
+                  key: howto
+              rows:
+                - auth: Claims-based authorization
+                  description: |
+                    Claims-based authorization uses a pair of options to manage claims verification during authorization.
+                    These pairs can be based on scopes, audience, groups, or roles claims. 
+                    <br><br>
+                    Learn more about [claims-based auth pairs](/plugins/openid-connect/#claims-based-authorization) in the OpenID Connect plugin documentation.
+                  example: "[Claims-based authorization example](/plugins/openid-connect/examples/claims-based-auth/)"
+                  howto: "[OpenID Connect and Keycloak with claims-based authorization](/how-to/configure-oidc-with-claims-based-auth/)"
+                - auth: ACL-based authorization
+                  description: |
+                    You can integrate the OpenID Connect plugin with the [ACL plugin](/plugins/acl/), 
+                    which provides access control functionality in the form of allow and deny lists.
+                  example: "[ACL-based authorization example](/plugins/openid-connect/examples/acl-auth/)"
+                  howto: "[OpenID Connect and Keycloak with the ACL plugin](/how-to/configure-oidc-with-acl-auth/)"
+                - auth: Consumer-based authorization
+                  description: |
+                    You can use {{site.base_gateway}} [Consumers](/gateway/entities/consumer/) for authorization and to dynamically map claim values to Consumers. 
+                  example: "[Consumer authorization example](/plugins/openid-connect/examples/consumer-auth/)"
```

```suggestion
                  example: "[Consumer-based authorization example](/plugins/openid-connect/examples/consumer-auth/)"
```
For consistency with the others